pkg/operator/vttablet/mysqlctld.go

```diff
@@ -29,6 +29,7 @@ const (
 	vtRootInitScript = `set -ex
 mkdir -p /mnt/vt/bin
 cp --no-clobber /vt/bin/mysqlctld /mnt/vt/bin/
+cp --no-clobber $(command -v mysqlbinlog) /mnt/vt/bin/ || true
```

Comment:
The directory `/mnt/.../` is shared across all the containers in the pod I am assuming? In which case, this line would resolve what you wrote in the PR's description:

> `/tmp` is not a shared mount point within the pod so when `mysqlbinlog` subsequently tries to read them from within the mysqld container it cannot find them in its container's `/tmp` directory and it fails with an error

Reply:
This is simply about copying the mysqlbinlog binary from the vitess/lite container image to the mysqlctld/vtbackup pod, as it looks like we'll need to keep that around in the lite image because the MySQL images do not contain that binary and it's needed for PITR.